src/tests/builders/chains.ts

```diff
@@ -27,8 +27,8 @@ const rpcUriBuilder = (): IBuilder<RpcUri> => {
 
 const blockExplorerUriTemplateBuilder = (): IBuilder<BlockExplorerUriTemplate> => {
   return Builder.new<BlockExplorerUriTemplate>().with({
-    address: faker.finance.ethereumAddress(),
-    txHash: faker.string.hexadecimal(),
+    address: faker.internet.url({ appendSlash: false }),
+    txHash: faker.internet.url({ appendSlash: false }),
     api: faker.internet.url({ appendSlash: false }),
```

Comment:
All of these are urls so I fixed it here. Seems to be a coincidence that this didn't break any tests yet.